### ISSUE TITLE
Proposal: Add Go 1.13 error handling to Error section

### ISSUE THREAD

**Issue by ngfgrant** [mmcgrana/gobyexample#334]
Hi thanks for this super site. 

I am going through it line by line at the moment and wondered if it would be worth making an addition to the existing Error page? Specifically, the new `%w`, `Unwrap`, `Is` and `As` methods in `error`. I don't think the new methods should replace the existing content, only compliment it. 

I am happy to contribute the example should this idea be accepted.

**Comment by eliben** [comment]
I think it's a good idea in general, but the devil is in the details. Wrapping errors is a power use, but gobyexample tries to stick to simpler concepts. Detailed discussion can take over a concrete PR, I just wanted to provide some background.

**Comment by Omid-SH** [comment]
Hi, [This](https://github.com/Omid-SH/Web_Final_Issues/tree/main/%23334/fourthIssue) may be helpful.

**Comment by eliben** [comment]
@Omid-SH consider submitting a PR with a *simple* example to this repository. That will be easier to review.

**Comment by guettli** [comment]
Yes, the [page about errors](https://gobyexample.com/errors) could be improved.

I am missing sentinel errors. We use them a lot and use errors.Is() often.

The current state seems to be: Waiting for PR. I hope I can provide one during the next days.

**Comment by guettli** [comment]
I created PR #517 which adds modern error handling. Please take a look. Feel free to take the PR and adapt it the way you want. Sometimes this is easier than telling me what I should change.
